src/main/kotlin/org/treeWare/proto3/aux/Proto3MetaModelMap.kt

```diff
@@ -1,6 +1,12 @@
 package org.treeWare.proto3.aux
 
-class Proto3MetaModelMap(val path: String) {
+class Proto3MetaModelMap(
+    val path: String?,
+    val imports: List<String>?,
```

Comment:
For the new parameters, add a comment explaining which level (meta-model, package, entity, field) they are applicable at.